test/integration.test.ts

```diff
@@ -284,6 +284,70 @@ const testCases: {
       expect(stderr).toContain(log)
     },
   },
+  {
+    name: 'external-entry',
+    args: ['--external', 'external-entry'],
+    async expected(dir) {
+      const distFiles = [
+        join(dir, './dist/index.cjs'),
+        join(dir, './dist/index.mjs'),
+        join(dir, './dist/server.cjs'),
+        join(dir, './dist/server.mjs'),
+      ]
+
+      expect(await fs.readFile(distFiles[0], 'utf-8')).toContain(
+        `Large piece of code`,
+      )
+      expect(await fs.readFile(distFiles[1], 'utf-8')).toContain(
+        `Large piece of code`,
+      )
+      expect(await fs.readFile(distFiles[2], 'utf-8')).not.toContain(
+        `Large piece of code`,
+      )
+      expect(await fs.readFile(distFiles[3], 'utf-8')).not.toContain(
+        `Large piece of code`,
+      )
+
+      expect(execSync('node ' + distFiles[2]).toString()).toBe(
```

Comment:
We don't have any execution test so far, I have a plan to make for most of test cases later. Would prefer to remove this for now

Reply:
I added it because I'm pretty sure `./index` is not a valid import specifier in Node 20.

Reply:
this test case is for `['--external', 'external-entry']`, which is already covered. should be able to support now. I would prefer to test the output content instead of exeuting the js bundle. The "./index" will be bundled into a different file